meta/documents/user_guide/en/obtaining_credentials.md

```diff
@@ -0,0 +1,44 @@
+# Wayfair plugin: Getting API Credentials
+
+In order for the Wayfair plugin to connect to Wayfair's systems, you need to provide your unique API credentials.
+
+To receive the API credentials for you organization, follow these steps:
+
+## 1. Send an Email to Wayfair
+
+Send an email to ERPSupport@wayfair.com to request assistance:
+
+- Subject : Access to Plentymarkets plugin / "Name of your company" (SuID)
+- Body:
+    - Contact information
+    - Descriptions of your organizations' plans and needs
+    - Any important dates regarding Plentymarkets >> Wayfair integrations.
+
+You will promptly receive a response containing these details:
+- Confirmation of access to
+- Supplier ID(s)

```

Comment:
From a supplier's perspective are they receiving "Warehouse" IDs?

Reply:
Great question.
@dFed17 @FelixEkwuemeC @umarjikaranisho please assist!

Reply:
I think that makes more sense - in PM, they're named `warehouses` correct?

Reply:
I'm going to leave it as `Supplier ID` due to our UI:
![image](https://user-images.githubusercontent.com/44010975/87829464-2363c700-c84d-11ea-850b-b2526abc14ea.png)